src/pages/playerScreen/index.js

```diff
@@ -0,0 +1,3 @@
+import PlayerScreen from './PlayerScreen';
```

Comment:
Player screen can be deleted as we planned to open player in fullscreen mode only